### DISCUSSION TITLE
dependent filter fields

### DISCUSSION THREAD

**Discussion by irvan-k** [carltongibson/django-filter#1408] [Q&A] [answered]
hi i want to have dependent filter fields for selecting city and then Neighborhood

**Comment by carltongibson** [comment, marked as answer]
You’d JS for the form, and the implement `clean()` to [validate across fields](https://docs.djangoproject.com/en/3.2/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other).
